packages/IMAPClient-UI.package/ICAddressBookDialog.class/instance/buildAddressBookEntryList..st

```diff
@@ -0,0 +1,17 @@
+toolbuilder
+buildAddressBookEntryList: builder
+	
+	^ builder pluggableTreeSpec new
```

Comment:
Why is this a tree? We do not have a hierarchy of mail addresses, do we? 
Is there something like a list spec?

Reply:
There is, but it does not fulfill our requirements. We used it then like the list of e-mails does.